packages/seacas/libraries/ioss/src/fmt/format-inl.h

```diff
@@ -1061,7 +1079,9 @@ namespace internal {
   template <typename T>
   int format_float(T value, int precision, float_specs specs, buffer<char> &buf)
   {
+#if !(defined(__INTEL_COMPILER) && __INTEL_COMPILER < 1800)
     static_assert(!std::is_same<T, float>(), "");
```

Comment:
Huh, can you actually convert an instance of `is_same` to `bool`?  That doesn't look right.  I would write this:
```cpp
static_assert(! std::is_same<T, float>::value, "T is not float");
```

Reply:
Thanks.  Looks like `std::is_same<...>()` returns a bool as of C++17, but for C++14 or earlier, the `::value` is needed.  I will fix and let lib::fmt project know.

Reply:
@gsjaardema Huh, you're right!  The object instance is convertible to `bool` (which is what you're relying on) as of C++11, and it has `bool operator()` [as of C++14](https://en.cppreference.com/w/cpp/types/is_same).  I would prefer `std::is_same<U, `V>::value` for C++11 code or `std::is_same_v<U, V>` for C++17 code, but that's correct syntax.